lib/vtls/vtls.c

```diff
@@ -885,6 +882,14 @@ static int multissl_init(void)
   return 1;
 }
 
+static CURLcode multissl_random(struct Curl_easy *data,
+                                unsigned char *entropy, size_t length)
+{
+  if(multissl_setup(NULL))
+    return CURLE_FAILED_INIT;
+  return Curl_ssl->random(data, entropy, length);
```

Comment:
This might need to be checked like if(Curl_ssl->random) if it's possible that a backend can be without a random